CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## (Unreleased)
 
-## 1.16.0 (April 29, 2024)
+## 1.16.0 (2024-04-29)
```

Comment:
This header is generated automatically, currently using the format defined in the [terraform changelog specification](https://developer.hashicorp.com/terraform/plugin/best-practices/versioning#version-headers) and used by other providers ([aws](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md)). Let me know if you see the need for this adjustment, we would need to adjust the script that creates this header during the release process.